benchmarks/DeepLearning/Layers/FFN/CMakeLists.txt

```diff
@@ -0,0 +1,132 @@
+add_custom_command(
+  OUTPUT
+  ${BUDDY_BENCHMARK_DEEP_LEARNING_DIR}/Layers/FFN/forward.mlir
+  ${BUDDY_BENCHMARK_DEEP_LEARNING_DIR}/Layers/FFN/subgraph0.mlir
+  COMMAND python ffn.py
+  WORKING_DIRECTORY ${BUDDY_BENCHMARK_DEEP_LEARNING_DIR}/Layers/FFN
```

Comment:
This cause error:
```
ninja: error: mkdir(/Layers): Permission denied
```
Please refer to the code [here](https://github.com/buddy-compiler/buddy-benchmark/blob/main/benchmarks/DeepLearning/Models/LeNet/CMakeLists.txt#L1)